__tests__/components/editor/InputListLOC.test.js

```diff
@@ -95,4 +101,9 @@ describe('<InputList />', () => {
   it('sets the formData store with the total number or objects sent to "selected" when a change event happens', () => {
     expect(mockFormDataFn.mock.calls.length).toBe(2)
   })
+
+  it('displays a browser alert if the lookupConfig is undefined', () => {
+    expect(global.alert.mock.calls.length).toBeGreaterThanOrEqual(1)
```

Comment:
Hmm, is the number of call times not deterministic?

Reply:
I moved the test to the top and then restored the mock, and now it it just a single call.

Reply:
excellent